oslabs-beta/shear
# Jk/cloud formation

## Summary
Changes to CloudFormation template, allowing AWS EIP association with EC2 instance.

## Related Issue
N/A

## Type of Change


## Testing Steps
npm run deploy in terminal
check your cloudformation stack
EC2 instance should have an assigned EIP.


## Checklist:
